src/tablib/formats/__init__.py

```diff
@@ -134,7 +116,13 @@ def formats(self):
     def get_format(self, key):
         if key not in self._formats:
             if key in uninstalled_format_messages:
-                raise UnsupportedFormat(uninstalled_format_messages[key])
+                uninstalled_format_messages[key].update(key=key)
```

Comment:
Instead of modifying a global, which is not so nice, I'd suggest to add `key=key` to the `format(...)`.

Reply:
Thanks, updated in 2b364fcc20c7d07ebdbe0b932e7d2fbbcb3eb860.